mlbgame/info.py

```diff
@@ -46,6 +46,88 @@ def team_info():
     return output
 
 
+def important_dates(year):
+    """Returns a dictionary of important dates"""
+    output = {}
+    data = mlbgame.data.get_important_dates(year)
+    important_dates = etree.parse(data).getroot().\
+        find('queryResults').find('row')
+    try:
+        for x in important_dates.attrib:
+            output[x] = important_dates.attrib[x]
+    except AttributeError:
+        raise ValueError('Unable to find important dates for {}.'.format(year))
+    return output
+
+
+class ImportantDates(mlbgame.object.Object):
+    """Holds information about important MLB dates and other info.
+    Properties:
+        all_star_date
+        all_star_sw
+        file_code
+        first_date_2ndh
+        first_date_seas
+        games
+        games_1sth
+        games_2ndh
+        last_date_1sth
+        last_date_seas
+        name_abbrev
+        name_full
+        name_short
+        org_code
+        org_type
+        organization_id
+        parent_abbrev
+        parent_org
+        playoff_games
+        playoff_points_sw
+        playoff_rounds
+        playoff_sw
+        playoff_teams
+        playoffs_end_date
+        playoffs_start_date
+        point_values
+        split_season_sw
+        wildcard_sw
+        wildcard_teams
+        year
+"""
+    def dtfmt(self, my_date):
```

Comment:
Slight simplification if you want: this can be a function in the module instead of a method on this class. Then you can just refer to it as `dtfmt` when calling it below instead of `self.dtfmt`.